scripts/minikube-minishift-setup-env.sh

```diff
@@ -6,51 +6,48 @@ shout() {
   set -x
 }
 
-# Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
-export HOME=`pwd`/home
-export GOPATH="`pwd`/home/go"
-export GOBIN="$GOPATH/bin"
-mkdir -p $GOBIN
 # This si one of the variables injected by ci-firewall. Its purpose is to allow scripts to handle uniqueness as needed
 SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
 
-# Add GOBIN which is the bin dir we created earlier to PATH so any binaries there are automatically available in PATH
-export PATH=$PATH:$GOBIN
+setup_operator() {
+  SETUP_OPERATORS="./scripts/configure-cluster/common/setup-operators.sh"
 
-# Prep for integration/e2e
-shout "Building odo binaries"
-make bin
+  # The OLM Version
+  export OLM_VERSION="v0.17.0"
+  # Enable OLM for running operator tests
+  curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/$OLM_VERSION/install.sh | bash -s $OLM_VERSION
 
-# copy built odo to GOBIN
-cp -avrf ./odo $GOBIN/
+  # Create Operators for Operator tests
```

Comment:
```suggestion
  # Install Operators for Operator tests
```
Being pedantic here, but we aren't creating any Operators, we are just using them. 😉

Reply:
😅 will update this too